package.json

```diff
@@ -3,47 +3,89 @@
+    "redux-thunk": "^2.2.0",
+    "smooth-scroll": "^0.1.4"
   },
   "devDependencies": {
-    "@jokeyrhyme/eslint-config-es2015": "^1.0.0",
-    "@jokeyrhyme/eslint-config-es5": "^1.0.0",
-    "babel-core": "^5.8.25",
-    "babel-eslint": "^4.1.4",
-    "babel-loader": "^5.3.2",
-    "clean-webpack-plugin": "^0.1.4",
-    "copy-webpack-plugin": "^0.2.0",
-    "css-loader": "^0.21.0",
-    "eslint": "^1.8.0",
-    "eslint-config-semistandard": "^5.0.0",
-    "eslint-config-standard": "^4.4.0",
-    "eslint-plugin-standard": "^1.3.1",
-    "file-loader": "^0.8.4",
-    "firebase-tools": "^2.1.0",
-    "htmllint-cli": "0.0.4",
-    "imports-loader": "^0.6.5",
-    "node-sass": "^3.3.3",
-    "sass-lint": "^1.3.1",
-    "sass-loader": "^3.0.0",
-    "style-loader": "^0.13.0",
-    "url-loader": "^0.5.6",
-    "webpack": "^1.12.2",
-    "webpack-dev-server": "^1.12.1"
+    "babel-core": "^6.24.1",
+    "babel-eslint": "^7.2.3",
+    "babel-loader": "^7.0.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-object-rest-spread": "^6.22.0",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-react-hmre": "^1.1.1",
+    "cross-env": "^5.0.0",
+    "css-loader": "^0.28.3",
+    "eslint": "^3.19.0",
+    "eslint-config-airbnb": "^15.0.1",
+    "eslint-config-prettier": "^2.1.1",
+    "eslint-plugin-import": "^2.3.0",
+    "eslint-plugin-jsx-a11y": "^5.0.3",
+    "eslint-plugin-prettier": "^2.1.1",
+    "eslint-plugin-react": "^7.0.1",
+    "file-loader": "^0.11.1",
+    "firebase-tools": "^3.9.0",
+    "husky": "^0.13.3",
+    "lint-staged": "^3.5.0",
+    "node-sass": "^4.5.3",
+    "prettier": "^1.3.1",
+    "remove-flow-types-loader": "^1.0.0",
+    "sass-lint": "^1.10.2",
+    "sass-loader": "^6.0.5",
+    "style-loader": "^0.18.1",
+    "stylefmt": "^6.0.0",
+    "stylelint": "^7.10.1",
+    "stylelint-config-standard": "^16.0.0",
+    "transform-loader": "^0.2.4",
+    "url-loader": "^0.5.8",
+    "webpack": "^2.6.1",
+    "webpack-dev-server": "^2.4.5"
   },
   "scripts": {
-    "build": "webpack",
-    "check:node": "check-node-version --node \">=7.6.0 <8\"",
-    "dev": "webpack-dev-server --devtool sourcemaps --progress --colors --hot --inline --content-base build",
-    "fix_js": "eslint --fix src/scripts",
-    "lint_html": "node_modules/.bin/htmllint build/**/*.html",
-    "lint_js": "eslint src/scripts",
-    "lint_sass": "node_modules/.bin/sass-lint -c sass-lint.yml -v",
-    "lint": "npm run lint_html && npm run lint_sass && npm run lint_js",
-    "test": "npm run lint",
-    "deploy": "npm run build && firebase deploy"
+    "upgrade": "yarn upgrade-interactive",
+    "deploy": "yarn run build && firebase deploy --token \"<TOKEN>\"",
+    "test": "yarn check && npm run lint",
+    "start": "webpack-dev-server",
+    "prod": "cross-env NODE_ENV=production webpack-dev-server --env.prod=true",
```

Comment:
prod, build and start should probably not come in yet. Bring the old webpack commands from the lhs back.